fault/verilator_target.py

```diff
@@ -534,6 +557,19 @@ def run(self, actions, verilator_includes=None, num_tests=0,
         with open(driver_file, "w") as f:
             f.write(src)
 
+        # if use kratos, symbolic link the library to dest folder
+        if self.use_kratos:
+            from kratos_runtime import get_lib_path
+            lib_name = os.path.basename(get_lib_path())
+            dst_path = os.path.abspath(os.path.join(self.directory, "obj_dir",
+                                                    lib_name))
+            if not os.path.isfile(dst_path):
+                os.symlink(get_lib_path(), dst_path)
+            # add ld library path
+            env = {"LD_LIBRARY_PATH": os.path.dirname(dst_path)}
```

Comment:
Same comment as above w.r.t. to macos (should either raise an error or we can just try setting DYLD_LIBRARY_PATH and hope that it works, travis does have a macos build so we could try running it on that).

Reply:
I don't have runtime build as a wheel for macos yet. I will debug with travis and once it's working I will and a separate PR.